target_builder/build_Windows-x86_64_rootfs

```diff
@@ -288,6 +288,7 @@ make_wsl_rootfs() {
 	ln -s usr/bin bin
 	ln -s usr/lib lib
 	ln -s usr/lib64 lib64
+	echo 'root:x:0:0:root:/root:/bin/sh' > etc/passwd
```

Comment:
Miss `/etc/shadow`?

Reply:
`shadow` is a file which contains the password information for the system's accounts and optional aging information.

So far, we have no program that needs to read shadow, and we do not need users to `login` to the ovm system, cause the wsl2 init (PID 1) process directly runs `/bin/sh` instead of `/bin/login`

Reply:
Okay